src/libraries/Common/src/Interop/Browser/Interop.JavaScript.HostObject.cs

```diff
@@ -0,0 +1,33 @@
+using System;
```

Comment:
Lots of these are missing the license header

Reply:
Added in previous commit.